internal/converter/converter.go

```diff
@@ -191,17 +204,23 @@ func (c *Converter) convertEnumType(enum *descriptor.EnumDescriptorProto, conver
 			_, valueDescription = c.formatTitleAndDescription(nil, src)
 		}
 
+		valueName := value.GetName()
+
+		if converterFlags.EnumsTrimPrefix {
+			valueName = strings.TrimPrefix(valueName, enumNamePrefix)
+		}
```

Comment:
It would be nice to have some descriptive comments around this. I know we understand what is being done right now, but next year someone may need the help when working on this code (I even struggle with my own code after a while, particularly in these big complex files with lots of branching logic).

Reply:
Sure thing. I added one where `enumNamePrefix` variable is defined, but missed this one.

I fixed original commit (to avoid additional ones).